LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Open Source Student Network
```

Comment:
As FixMe is a product the copyright should be a range i.e. "2018-2019"

Reply:
Updated in #143